### PR TITLE
examples/data-collect: Remove fsf 

### PR DESCRIPTION
**Pull request by rbudai98**

### CODE REVIEW

**Review by dNechita — APPROVED** [review]

File: examples/data_collect/main.cpp [hunk 1]
```diff
@@ -790,29 +575,6 @@ int main(int argc, char *argv[]) {
 #else
         char out_file[MAX_FILE_PATH_SIZE];
 
-        if (!fsf_flag) {
-            snprintf(out_file, sizeof(out_file), "%s/%s_frame_%s_%05u.bin",
```
Comment (SeptimiuVana) [inline examples/data_collect/main.cpp:794]: I think we need to keep this if we want to save the frames without multi threading.

Comment (rbudai98) [reply examples/data_collect/main.cpp:794]: Will replace it without the if condition

**Review by dNechita — CHANGES_REQUESTED** [review]
why are changes to the tool/tof_lib_gen included in the PR?
Also, the title & description of the commit has nothing to do with intention of this PR.

**Comment by rbudai98** [comment]
v3:
* commit message and title corrected
* removed unrelated changes

**Review by dNechita — APPROVED** [review]

**Pull request merged**